spark/src/test/scala/org/apache/spark/sql/delta/managedcommit/CommitOwnerClientImplSuiteBase.scala

```diff
@@ -204,13 +204,7 @@ trait CommitOwnerClientImplSuiteBase extends QueryTest
       }
       assert(e.getMessage === "Commit version 0 must go via filesystem.")
       writeCommitZero(logPath)
-      var expectedVersion = -1
-      if (tableCommitOwnerClient.commitOwnerClient.isInstanceOf[DynamoDBCommitOwnerClient]) {
-        // DynamoDBCommitOwnerClient stores attemptVersion as the current table version when
-        // registerTable is called.
-        expectedVersion = 0
-      }
-      assert(tableCommitOwnerClient.getCommits() == GetCommitsResponse(Seq.empty, expectedVersion))
+      assert(tableCommitOwnerClient.getCommits() == GetCommitsResponse(Seq.empty, -1))
```

Comment:
Could we add test to validate that the spark config overrides are correctly used by DynamoDBCommitOwnerClient.

Reply:
Added a test for this: https://github.com/delta-io/delta/pull/3130/files#diff-dc7d3ff136c9f5338c9524783e00375fc251c212230e0b366581b1e5b53b8387R242